roles/template_service_broker/tasks/install.yml

```diff
@@ -43,16 +48,18 @@
 
 - name: Apply template file
   shell: >
-    {{ openshift_client_binary }} process -f "{{ mktemp.stdout }}/{{ __tsb_template_file }}"
+    {{ openshift_client_binary }} process --config={{ mktemp.stdout }}/admin.kubeconfig
+    -f "{{ mktemp.stdout }}/{{ __tsb_template_file }}"
     --param API_SERVER_CONFIG="{{ config['content'] | b64decode }}"
     --param IMAGE="{{ template_service_broker_prefix }}{{ template_service_broker_image_name }}:{{ template_service_broker_version }}"
     --param NODE_SELECTOR={{ template_service_broker_selector | to_json | quote }}
-    | {{ openshift_client_binary }} apply -f -
+    | {{ openshift_client_binary }} apply --config={{ mktemp.stdout }}/admin.kubeconfig -f -
 
 # reconcile with rbac
 - name: Reconcile with RBAC file
   shell: >
-    {{ openshift_client_binary }} process -f "{{ mktemp.stdout }}/{{ __tsb_rbac_file }}" | {{ openshift_client_binary }} auth reconcile -f -
+    {{ openshift_client_binary }} process --config={{ mktemp.stdout }}/admin.kubeconfig -f "{{ mktemp.stdout }}/{{ __tsb_rbac_file }}" 
```

Comment:
@bparees the tox test is failing - wonder if it's the trailing space on this line (change your editor settings? :)